src/main/DAL/MetaQuestion/MetaQuestion.js

```diff
@@ -5,25 +5,14 @@ const {META_QUESTION_TYPES} = require("../../Enums");
  * Defines a 'MetaQuestion' model within the given database
  */
 function defineMetaQuestionModel(sequelize) {
-    if (sequelize.modelManager.models.some(model => model.name === 'MetaQuestion')) {
-        return sequelize.models.MetaQuestion;
-    }
-    class MetaQuestion extends Model {}
-    MetaQuestion.init({
-        stem: {
-            type: DataTypes.STRING,
-            allowNull: false
-        },
-        questionType: {
-            type: DataTypes.ENUM,
-            values: Object.values(META_QUESTION_TYPES),
-            allowNull: false
-        }
-    }, {
-       sequelize,
-       modelName: 'MetaQuestion',
-    });
-    return sequelize.models.MetaQuestion;
+    return sequelize.modelManager.models.some(model => model.name === 'MetaQuestion') ?
```

Comment:
we said that metaquestion should have id

Reply:
they do have an ID
By not defining any Primary-Key, sequelize gives an enumerated id column to the table.

Reply:
check out the tests